### DISCUSSION TITLE
Demo Session: Merging made fast and easy - Demystiying GitHub's merge queue

### DISCUSSION THREAD

**Discussion by thejcannon** [community/community#143095] [Other Feature Feedback, Questions, & Ideas]
:wave: Howdy!

If you had questions or wanted to chat regarding the demo session on merge queues, now is your chance!

https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715220398236001jV6C

**Comment by thejcannon** [comment]
(If you were sitting in the front row and taking a video of the whole thing, I'd love if you could reach out. I'd be so delighted to have a recording of it!)

**Reply by sajangurung** [reply]
Great talk @thejcannon, would you be able to share the presented slides here by any chance?
